i18next-parser.config.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
   lineEnding: 'auto',
   // Control the line ending. See options at https://github.com/ryanve/eol
 
-  locales: ['en', 'pt', 'de', 'fr', 'ru', 'pl'],
+  locales: ['en', 'pt', 'de', 'fr', 'ru', 'pl', 'tr'],
```

Comment:
don't forget to add the `du` for dutch here as well 👍🏽

Reply:
Added `nl` because that's what is commonly being used. I know it doesn't make sense. :smile: